config.js

```diff
@@ -637,6 +637,9 @@ config.INLINE_MAX_SIZE = 4096;
 // Object SDK bucket cache expiration time
 config.OBJECT_SDK_BUCKET_CACHE_EXPIRY_MS = 60000;
 
+// Object SDK bucket_namespace_cache allow stat of the config file
+config.ALLOW_BUCKET_NS_CACHE_STAT = true;
```

Comment:
@shirady regarding tests - 
1. Can you enable the ceph tests that should work now due to this fix
2. Can you think of unit/integration tests for checking it?

Reply:
Regarding the Ceph tests on NSFS - I think that running the tests with a higher number of forks makes the logs less readable (and in case one made a passing test to get failed we want it will be easiest to understand what happened.

I run the `make test-nsfs-cephs3 CONTAINER_PLATFORM=linux/arm64` with codes change:
1) Change the mount point
```diff
- export CEPH_TEST_LOGS_DIR=/logs/ceph-nsfs-test-logs
+ export CEPH_TEST_LOGS_DIR=./logs/ceph-nsfs-test-logs
```
 To avoid the error:

>  docker run  --privileged --user root --name noobaa_"aaeeb19"_"42" --env "SUPPRESS_LOGS=""" -v <>/noobaa-core/logs:/logs "noobaa-tester" "./src/test/system_tests/ceph_s3_tests/run_ceph_nsfs_test_on_test_container.sh"
> docker: Error response from daemon: error while creating mount source path '<>/noobaa-core/logs': chown <>/noobaa-core/logs: permission denied.
> ERRO[0000] error waiting for container:
> make: *** [test-nsfs-cephs3] Error 126

(Note: `<>` removed the internal path)


2) Set a higher number of forks
```diff
- config='{"ALLOW_HTTP":true}'
+ config='{"ALLOW_HTTP":true, "ENDPOINT_FORKS":16}'
```

I ran it twice and saw there are tests that failed (In those tests there is a versioning status change):
1) On current master:

> nsfs: merged config.json= { ALLOW_HTTP: true, ENDPOINT_FORKS: 16 }
> ...
> Nov-17 6:47:39.255 [test_ceph_s3/275]  [INFO] CONSOLE:: CEPH TEST SUMMARY: Suite contains 801, ran 262 tests, Passed: 243, Skipped: 16, Failed: 3
> ...
> Nov-17 6:47:39.255 [test_ceph_s3/275] [ERROR] CONSOLE:: CEPH TEST FAILED TESTS SUMMARY: 3 failed tests
> s3tests_boto3/functional/test_s3.py::test_versioning_obj_plain_null_version_overwrite
> s3tests_boto3/functional/test_s3.py::test_versioning_obj_suspend_versions
> s3tests_boto3/functional/test_s3.py::test_versioning_obj_suspended_copy

2) On this branch (with code changes):

> nsfs: merged config.json= { ALLOW_HTTP: true, ENDPOINT_FORKS: 16 }
> ...
> Nov-17 8:09:08.003 [test_ceph_s3/276]  [INFO] CONSOLE:: CEPH TEST SUMMARY: Suite contains 801, ran 262 tests, Passed: 246, Skipped: 16, Failed: 0

Reply:
Regarding the idea to run with more forks - I added it (in the last 2 commits).

Reply:
@nimrod-becker @liranmauda 
Shira and I are discussing if we should have 2 runs of ceph tests, one with a single process and one with multiple processes. WDYT? since we have limited resources

Reply:
The decision is to change the existing CI to run with 2 forks.